### DISCUSSION TITLE
DataTable: Arial-label issues

### DISCUSSION THREAD

**Discussion by xaxoxavi** [primefaces/community#1240] [PrimeVue] [answered]
Some elements (buttons or form elements) generated by the DataTable component do not have an 'aria-label' attribute.
Here is the list of the ones I spotted:

Same issue resolved in [primereact](https://github.com/primefaces/primereact/issues/3263)

(1) The filter button generated by property filterDisplay='menu
(2) The paginator dropdown menu to select the number of rows to display. This element generates a 'p-hidden-accessible' div with an input element (with role="textbox") in it, but this input has no aria-label attribute. It also generates a 'p-dropdown-trigger' div with the role 'button' and this div has no aria-label attribute
(3) The radio buttons and checkboxes generated in selection mode

(1)
![image](https://github.com/primefaces/community/assets/17217367/75167801-55c7-4eca-b39f-0330cf03d050)
(2)
![image](https://github.com/primefaces/community/assets/17217367/e6cbb8ac-b1b4-4d50-b8ae-92f804006900)
(3.1)
![image](https://github.com/primefaces/community/assets/17217367/655083ba-94fd-45d3-ae05-cfdac0ec4da1)
(3.2)
![image](https://github.com/primefaces/community/assets/17217367/38acdd2d-d1bb-4145-baf4-fe01ad68c8a3)

**Comment by tugcekucukoglu** [comment, marked as answer]
Thanks for the feedback. Could you create an issue about this?

Thank you.

**Reply by xaxoxavi** [reply]
Ok
